docs/source/tutorials/odsc_tutorial/comparison.rst

```diff
@@ -0,0 +1,81 @@
+Comparing Different HPO Methods
+===============================
+
+We have learned about different methods for hyperparameter tuning:
```

Comment:
Minor nit: I think this is the first time that these methods are introduced in the tutorial? So at this point this is new material for the readers? 

(still reading through the whole tutorial, my meta-comment is that this is great, really well-written. All my comments are minor optional nits so far)

Reply:
Yes, there are slides where these methods are all introduced.

Have you looked at the slides? Whenever I switch to the reST, I inserted a slide with a screenshot.

Reply:
Ah ok. I was reviewing this solely from the perspective of external readers who will find the tutorial on our documentation website -- all my comments are from that perspective.